src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -82,7 +80,7 @@ private static string GetTriggerInputName(FunctionMetadata functionMetadata, str
                 triggerName = triggerBinding.Name;
             }
 
-            return triggerName ?? defaultInputParameterName;
+            return triggerName;
```

Comment:
Should we ever return `null` from this? Should we be failing if we can't resolve the name?

By the way, now we should be able to use `triggerBinding?.Name` or return straight from `functionMetadata.Bindings.FirstOrDefault(b => b.IsTrigger)?.Name`